tests/test-cl-image.cpp

```diff
@@ -543,7 +547,28 @@ int main (int argc, char *argv[])
         denoise_config.threshold[1] = 0.05;
         denoise_config.gain = 0.6;
         denoise->set_denoise_config (denoise_config);
-        XCAM_ASSERT (denoise.ptr ());
+        break;
+    }
+    case TestHandlerImageWarp: {
+        image_handler = create_cl_image_warp_handler (context);
+        SmartPtr<CLImageWarpHandler> warp = image_handler.dynamic_cast_ptr<CLImageWarpHandler> ();
+        XCAM_ASSERT (warp.ptr ());
+        XCamDVSResult warp_config;
+        xcam_mem_clear (warp_config);
+        warp_config.frame_id = 1;
+        warp_config.frame_width = width;
+        warp_config.frame_height = height;
+        warp_config.valid = true;
+        warp_config.proj_mat[0] = 1.0f * cos(-6.0f * 3.1415926 / 180.0f);
```

Comment:
how about add variables to read easily
float rotation_angle = 6.0f;
float rotation_offset[2] ={192.0f, 108.0f};
...